bin.js

```diff
@@ -35,6 +35,14 @@ program.command('exchanges.total').action(async () => {
 	exchanges.total().then(console.log);
 });
 
+program
+	.command('exchanges.aggregate')
+	.option('-e, --entity <value>', 'The type of total - dailyTotals, fifteenMinuteTotals', 'dailyTotals')
```

Comment:
perhaps just `1d` or `15m` as the granularity and you map that to the entity name?

Reply:
ok cool 👍